posthog-node/src/feature-flags.ts

```diff
@@ -427,10 +419,26 @@ class FeatureFlagsPoller {
         this.featureFlagsByKey = {}
         this.groupTypeMapping = {}
         this.cohorts = {}
-        this.loadedSuccessfullyOnce = false
+        this.loadedSuccessfullyOnce = true
```

Comment:
This line here is the main fix, correct? Might be worth a comment on why this is set to `true` here even though we're in a `402` status. I understand now, but at first, I thought this was a mistake.

Reply:
actually, this line was not the fix that we needed, and I've since removed it.  There was some confusion about the original cause of this error – originally we thought it was from quota limited responses (402s), but turns out it was actually that we were repeatedly fetching _rate_-limited responses (429s).

So, the changes I made to that end were:
- wrap the rate limiting errors in the same exponential backoff as 401 and 403s
- continue to do the same thing with 402s, which is clear the flags but _don't_ return the error.

See my latest diff for more context on all of this